etc/bento_dev.env

```diff
@@ -14,31 +8,53 @@ BENTOV2_PRIVATE_MODE=false
 BENTO_CBIOPORTAL_ENABLED=false
 # Feature switches end ------------------------------------------------
 
+# Root data directory on host. Bind volumes for services will be put
+# here as subdirectories and mounted into the relevant containers.
 BENTOV2_ROOT_DATA_DIR=~/bentov2/data
```

Comment:
I think for the dev env we could set it to ./data by default, if users want to use more than one bentoV2 on a machine the instances could point to the same data directory if they are not careful with this variable

Reply:
Yeah I agree, will do this
we will have to warn people upgrading but I will put something in the migration doc

Reply:
i guess actually no warning needed since local.env will work as-is but i put a note anyway